src/main.rs

```diff
@@ -33,6 +33,7 @@ enum SubCommand {
 }
 
 fn main() {
+    env_logger::builder().format_timestamp(None).init();
```

Comment:
Why do you add this? We use the syslog logger not env logger since stderr/out are not open.

Reply:
Only for debugging. Not needed on main.

Reply:
removed.